### ISSUE TITLE
Charakterystyczne zdjęcia miast

### ISSUE THREAD

**Issue by teklakct** [PHPers/website#54]
W założeniach mam by zamiast dziewczyn na kafelkach kalendarza dać charakterystyczne zdjęcia/grafiki/szkice dla poszczególnych miast.

Potrzebujemy znaleźć/przygotować odpowiednie pliki

**Comment by leafnode** [comment]
Źródła:
https://commons.wikimedia.org/wiki/Main_Page
https://www.flickr.com/ (z wyszukiwaniem po licencji)

Pamiętajcie o licencjach.

**Comment by l3l0** [comment]
Przykładowe fotki z Gdańska:

https://www.flickr.com/photos/altotemi/29727376311/in/photolist-MhUzhT-7gVVPj-89xSNY-uyteVy-ipjL2U-8nSNXy-cnTFN3-cmcYro-82fLzE-7yDz5q-aUqhMi-6LPMUK-6LTWTj-i8nici-6LPLoe-6LTWco-6LTVPU-8nSQhG-nFzfuv-chvkvh-8nSPbo-chviRG-6LPMjn-aUqhFR-8nPHC2-8nSSDC-8nPFHa-8nPH9a-8nST7J-8nPBVM-8nSToh-8nPGU8-6e2QZL-8nSRRs-6dXF2B-8nSNaL-8nPC8e-8nPCSK-8nSNL5-8nPBKg-8nSUVo-8nSPoS-8nPCpV-8nPGca-8nPD68-8nSUrG-8nSQGj-8nPKra-8nSUgG-8nSTMj

https://www.flickr.com/photos/altotemi/12015457703/in/photolist-jiLmXR-jiWU8E-jiLAEm-jiKUmB-jiKZtQ-oz3Seg-nYacGa-nW7RXY-nY3RjA-nY2poj-nzNC62-nqoX4F-jhekLa-nzNXhZ-oz3i3N-nfWu1t-ny3ZT5-H4Wg6H-jbSnk7-Chn3UT-BRXAwF-BRaFzt-jiL8Lk-XNVo23-Cksooz-ChmRYz-Bs89Gy-CniBWC-jacg8w-CnjHoN-BR7Mf2-Bs8AjL-ChkQwZ-nw7B3k-jiMxNC-G9qBmE-jiMhTB-Cf3S1Y-ja7LAD-jaa71A-Btzy44-BYQBb9-neBS48-qowVRt-ir6k9D-4icvWo-jiLm4s-GVAD6u-4i8pyx-nYeTJB

https://commons.wikimedia.org/wiki/Gda%C5%84sk#/media/File:Danzig-krantor03.jpg

**Comment by michailw** [comment]
Co powiecie na te?

Warszawa
========
Link: https://www.flickr.com/photos/filip_bramorski/14580701577/in/photolist-odrUAv-213bzT-xVXXv8-xWC3s2-wYX6RN-2kqeRW-cXroNo-bubGjZ-ceda9w-bRVyj6-dhY9zr-ddKhXZ-pPS51Y-bTs7Ce-bVm1Uc-c7dioY-t1bQXX-bkRhVi-V2dNnM-nRFCMM-aSV4Ni-NdnmkE-p8612E-bkRjeX-bzUSGz-87Nun4-oATLKe-do1MEh-22Drfh-8CNuDU-bkRi7i-2kqeRo-bVm4bx-5aqkY9-58pMPp-nRfuvK-cC3wwq-c7dkLW-4e7FXL-do1Ury-e24Zf7-ddJdEj-naZgpG-95bPE2-2wSU7n-do1SQf-nRFCLp-do1X6m-do1RvS-c7dfzW
![Creative Commons](https://c1.staticflickr.com/6/5567/14580701577_ac81f55c8b_b.jpg)

Trójmiasto
======
Link: https://www.flickr.com/photos/petrolly/18584014064/in/photolist-i2JCWq-onwhbc-ujcVVA-idn4zQ-Rf56eK-QRnkWo
![Creative Commons](https://c1.staticflickr.com/1/437/18584014064_92a714eb63_h.jpg)

Kraków
======
Link: https://www.flickr.com/photos/pedrocaetano/5060336981/in/photolist-8HaxTk-XJY6aa-Wt9tkW-p9YQQQ-iUM64n-5qeizA-5qoXfC-8KtDuq-8HyAXQ-8KtDij-hN7ruC-5q9JYR-8KqAmR-5mFw3J-s8ammm-hN6Mh4-erbKfa-5mFwd5-5qeizE-XSfDNT-5qeizo-fZTWMU-5q9JYH-8JxUNX-oNUzLC-8Hr6SG-SLFirT-qSQz6d-8GXo52-5qoXfy-8CTfbt-5q9JYz-8HdEUm-Xu59zS-5qeizU-5qeiA5-8JxV4v-5q9JYD-B88WY-5q9JZ2-8K7u1h-X8UbBW-iqbfp-5mFwiL-bGcF6D-4rgZ8V-ouAftz-5q9JYV-B88Yx-hN6qGr
![Creative Commons](https://c1.staticflickr.com/5/4131/5060336981_5fd856d682_b.jpg)

Bielsko - Biała
===============
![Public Domain](https://upload.wikimedia.org/wikipedia/commons/f/f7/Su%C5%82kowski_Castle%2C_Bielsko-Bia%C5%82a_3.jpg)

**Comment by teklakct** [comment]
@michailw it's legal to use it?

**Comment by michailw** [comment]
@teklakct Tak, jest to legalne - wybieram zdjęcia z licencją CC albo Public Domain. Z Gdańska wytniemy sobie żurawia (możesz zobaczyć na pull requescie próbkę)

**Comment by filipgolonka** [comment]
@michailw Kraków spoko :)

**Comment by michailw** [comment]
Poznań
=====
Link: https://commons.wikimedia.org/wiki/File:6_Poznan_015.jpg
![Creative Commons](https://upload.wikimedia.org/wikipedia/commons/4/49/6_Poznan_015.jpg)

**Comment by michailw** [comment]
Toruń
====
Link: https://commons.wikimedia.org/wiki/Toru%C5%84#/media/File:Torun01CenterWithWall_level_corr.jpg
![Creative commons](https://upload.wikimedia.org/wikipedia/commons/1/1a/Torun01CenterWithWall_level_corr.jpg)

**Comment by michailw** [comment]
Wrocław
======
Link: https://commons.wikimedia.org/wiki/File:Plac_solny_(Wroc%C5%82aw).jpg
![Public domain](https://upload.wikimedia.org/wikipedia/commons/c/ce/Plac_solny_%28Wroc%C5%82aw%29.jpg)

**Comment by dedik** [comment]
Poznań:

https://upload.wikimedia.org/wikipedia/commons/7/7d/Old_marketplace_and_city_hall_in_Pozna%C5%84.jpg

**Comment by michailw** [comment]
Olsztyn
======
Link: https://commons.wikimedia.org/wiki/File:Olsztyn_Katedra_noc.jpg
![Public domain](https://upload.wikimedia.org/wikipedia/commons/d/d9/Olsztyn_Katedra_noc.jpg)

**Comment by michailw** [comment]
Rzeszów
=======
Link: https://commons.wikimedia.org/wiki/File:P8100166.JPG
![Public domain](https://upload.wikimedia.org/wikipedia/commons/e/ef/P8100166.JPG)

**Comment by michailw** [comment]
Gliwice
=====
Link https://www.flickr.com/photos/eisenbahner/7561490938/in/photostream/
![Creative commons](https://c1.staticflickr.com/8/7133/7561490938_c92988af1b_b.jpg)
